packages/grafana-ui/src/components/Link/TextLink.tsx

```diff
@@ -27,7 +27,7 @@ interface TextLinkProps extends Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 't
   weight?: 'light' | 'regular' | 'medium' | 'bold';
   /** Set the icon to be shown. An external link will show the 'external-link-alt' icon as default.*/
   icon?: IconName;
-  children: string;
+  children: React.ReactNode;
```

Comment:
We need to change for the TextLink to work with `<Trans />`

Reply:
✅  LGTM 👍